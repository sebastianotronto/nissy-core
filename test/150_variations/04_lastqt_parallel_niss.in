lastqt
L2 R U' D (U' D')
