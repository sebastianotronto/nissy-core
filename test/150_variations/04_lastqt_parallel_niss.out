16
L2 R U D (U D)
L2 R U D (U D')
L2 R U D (U' D)
L2 R U D (U' D')
L2 R U D' (U D)
L2 R U D' (U D')
L2 R U D' (U' D)
L2 R U D' (U' D')
L2 R U' D (U D)
L2 R U' D (U D')
L2 R U' D (U' D)
L2 R U' D (U' D')
L2 R U' D' (U D)
L2 R U' D' (U D')
L2 R U' D' (U' D)
L2 R U' D' (U' D')
